builtin/providers/postgresql/provider.go

```diff
@@ -35,6 +35,12 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("POSTGRESQL_PASSWORD", nil),
 				Description: "Password for postgresql server connection",
 			},
+			"sslmode": &schema.Schema{
```

Comment:
Thank you for your answer !
I've just amended/pushed the modification.

P.S:
Franckly, this _delay_ is acceptable :)